pages/content/amp-dev/documentation/guides-and-tutorials/contribute/contribute-documentation/formatting.md

```diff
@@ -0,0 +1,275 @@
+---
+$title: Formatting guides & tutorials 
```

Comment:
Should this live in parallel to https://github.com/ampproject/amp.dev/blob/future/contributing/documentation.md#code-samples? We should merge these two.